<ui version="4.0" >
 <class>QueryWidgetClass</class>
 <widget class="QWidget" name="QueryWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sql Query</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="spacing" >
        <number>2</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit" />
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>268</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="exeBtn" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font" >
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text" >
            <string>Execute(^F9)</string>
           </property>
           <property name="flat" stdset="0" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="lineExeBtn" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font" >
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text" >
            <string>Exe. Line(^Enter)</string>
           </property>
           <property name="flat" stdset="0" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="selExeBtn" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font" >
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text" >
            <string>Exe. Selected(F9)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="GTableView" name="tableView" />
     <widget class="QPlainTextEdit" name="logEdit" />
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>GTableView</class>
   <extends>QTableView</extends>
   <header>GTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
